config/pricealert.yaml

```diff
@@ -20,6 +20,8 @@ sessions:
   binance:
     exchange: binance
     envVarPrefix: binance
+    #margin: true
+    futures: true
```

Comment:
👍

Reply:
add another config file example for the futures ?

Reply:
make this pricealert config clean, because it's for the pricealert strategy

Reply:
Sure, done!